bus-mapping/src/circuit_input_builder/input_state_ref.rs

```diff
@@ -1135,16 +1145,13 @@ impl<'a> CircuitInputStateRef<'a> {
                     false,
                     op,
                 );
-                let idx = if from_precompile_call {
-                    // prev step is callop or begin_tx.
-                    // the failed transfer should be reverted there.
-                    step_index - 1
+                if is_from_precompile_failure {
+                    exec_step.bus_mapping_instance.push(rev_op_ref);
```

Comment:
for callop, the rev_op_ref should be attached onto the CallopExecState or PrecompileExecState?

Reply:
pushed an update in https://github.com/scroll-tech/zkevm-circuits/pull/855/commits/b678e7057487728aa7c1988c9d8683529dd3ea37